tracetesting/transactions/test-ids.yaml

```diff
@@ -0,0 +1,15 @@
+type: Transaction
+spec:
+  id: TmAKdfV4g
+  name: Test IDs
+  env:
```

Comment:
This could also be named `variables`. It represents the env/vars initial values that will be passed to every test.
Note that this is a initial value, and can be overwritten by a test output with the same name.

This is not canonical, meaning that a test could create an output that is not defined here, and that output will be also passed to subsequent tests.